english/yoga/habit/yoga-habit.md

```diff
@@ -1,5 +1,5 @@
 It's hard to build a habit of doing yoga on a regular basis. Here are some ways
 to make it easier to get going:
 
-1. Just do it!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.